templates/zerver/help/index.md

```diff
@@ -91,7 +89,7 @@ as an **organization**.
 * [Rename a stream](/help/rename-a-stream)
 * Preview a stream (not implemented)
 * [Unsubscribe from a stream](/help/unsubscribe-from-a-stream)
-* [Change who can join a stream](/help/change-who-can-join-a-stream)
+* [Change the accessibility of a stream](/help/change-the-accessibility-of-a-stream)
```

Comment:
accessible |akˈsesəbəl| adjective
1 (of a place) able to be reached or entered: the town is accessible by bus | this room is not accessible to elderly people.